docs/reference/query-dsl/geo-polygon-query.asciidoc

```diff
@@ -4,8 +4,7 @@
 <titleabbrev>Geo-polygon</titleabbrev>
 ++++
 
-deprecated[7.12.0,"Use <<query-dsl-geo-shape-query>> instead,
-where polygons are defined in geojson or wkt."]
+deprecated:[7.12.0] Use <<query-dsl-geo-shape-query>> instead, where polygons are defined in GeoJSON or http://docs.opengeospatial.org/is/18-010r7/18-010r7.html[Well-Known Text (WKT)].
```

Comment:
Drive-by nit:

I think this should use the block format `deprecated::[7.12.0,...]` so it displays as an admon.

Reply:
Forgot to mention: Thanks for spotting and fixing this! ❤️

Reply:
Thanks @jrodewig! I didn't know that was possible, so appreciate the heads up. I'll make that change.